js/compiler/compiler_test.go

```diff
@@ -71,13 +72,26 @@ func TestTransform(t *testing.T) {
 		// assert.Equal(t, "test.js", srcmap.File)
 		// assert.Equal(t, "aAAA,SAASA,GAAT,CAAaC,CAAb,EAAgBC,CAAhB,EAAmB;AACf,WAAOD,IAAIC,CAAX;AACH;;AAED,IAAIC,MAAMH,IAAI,CAAJ,EAAO,CAAP,CAAV", srcmap.Mappings)
 	})
+
+	t.Run("double-arrow with sourceMap", func(t *testing.T) {
+		c.COpts.SourceMapEnabled = true
+		src, _, err := c.Transform("()=> true", "test.js", nil)
+		assert.NoError(t, err)
+		assert.Equal(t, `"use strict";
+
+() => true;
+//# sourceMappingURL=k6://internal-should-not-leak/file.map`, src)
+		// assert.Equal(t, 3, srcmap.Version)
+		// assert.Equal(t, "test.js", srcmap.File)
+		// assert.Equal(t, "aAAA,qBAAK,IAAL", srcmap.Mappings)
```

Comment:
Should we enable them?

Reply:
This is leftover (as are the rest of the similar commented checks) from when k6 parsed the sourcemap. But in reality we don't need to as k6 expects unparsed so we have no need to parse it into a struct and then check what the values are IMO